README.md

```diff
@@ -3,11 +3,61 @@
 > :warning: **Under Development**
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/openshift/managed-upgrade-operator)](https://goreportcard.com/report/github.com/openshift/managed-upgrade-operator)
-![Go Coverage](./coverage_badge.png)
+[![codecov](https://codecov.io/gh/openshift/managed-upgrade-operator/branch/master/graph/badge.svg)](https://codecov.io/gh/openshift/managed-upgrade-operator)
 [![GoDoc](https://godoc.org/github.com/openshift/managed-upgrade-operator?status.svg)](https://pkg.go.dev/mod/github.com/openshift/managed-upgrade-operator)
 [![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
 
+----
+
+The _Managed Upgrade Operator_ has been created for the [OpenShift Dedicated Platform](https://docs.openshift.com/dedicated/4/) (OSD) to manage the orchestration of automated in-place cluster upgrades.
```

Comment:
Do we want to add any note about the supported/tested platforms as well? (AWS/GCP/Azure etc?)

Reply:
OSD != Azure. We can assume OSD supported platforms  = AWS/GCP as per OCM interface capabilities. Trivial whether we specify this, I am not opposed to either. Not sure any other operator does it.